src/Swipeable.js

```diff
@@ -30,7 +31,8 @@ const Swipeable = React.createClass({
     return {
       flickThreshold: 0.6,
       delta: 10,
-      preventDefaultTouchmoveEvent: true
+      preventDefaultTouchmoveEvent: true,
+      component: 'div'
```

Comment:
should we name this `nodeName` instead or will ya'll accept a React component in here? If so, we should update the propTypes.